com.unity.render-pipelines.high-definition/Documentation~/Upgrading-from-2020.1-to-2020.2.md

```diff
@@ -165,7 +166,12 @@ to:
 For example, the call in the Lit shader has been updated to:
 `float4 preLD = SampleEnv(lightLoopContext, lightData.envIndex, R, PerceptualRoughnessToMipmapLevel(preLightData.iblPerceptualRoughness), lightData.rangeCompressionFactorCompensation, posInput.positionNDC);`
 
-## raytracing
+From 10.x, a new optimization have been introduce for [Planar Reflection Probe](Planar-Reflection-Probe.md). It required to sample the environment map at the mip 0 with the help of the LightData.roughReflections parameter to be effective in custom shader.
```

Comment:
From 10.x, HDRP includes a new optimization for [Planar Reflection Probes](Planar-Reflection-Probe.md). To sample the probe's environment map at mip level 0 in a custom shader, you now must use LightData.roughReflections.


This is how I interpreted this, is it correct?